packages/flutter/test/widgets/default_text_editing_actions_test.dart

```diff
@@ -73,7 +73,7 @@ void main() {
       ),
     ));
 
-    // Focus on the EditableText, which is a TextEditingActionTarget.
```

Comment:
I thought I removed this file, since `TextEditingActionTarget` is gone?

Reply:
So did I but I also noticed that text_editing_action_target.dart was still in the repo unused!  I removed it in this PR.  Maybe it accidentally came back in a merge or something and we missed it?

I went ahead and deleted this test file too now.